src/js/middleware.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+// Base grade for express middleware modules...
+//
+// The express module will automatically attempt to wire in anything with this gradeName into itself
+
+var fluid = fluid || require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.express.middleware");
+
+gpii.express.middleware.complainAboutMissingFunction = function(){
```

Comment:
rerun the jshint task since I believe that our standard configuration should complain about the whitespace here, if it doesn't, there might be a problem with the task configuration